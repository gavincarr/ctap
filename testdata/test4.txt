1..14
not ok - csv is NOT regular
ok - header is as expected
ok - row_count >= 100 (125)
ok - columns.a.fill_rate == 100
ok - columns.b.fill_rate >= 33 (34.33)
