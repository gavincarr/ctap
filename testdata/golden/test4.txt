[38;2;153;153;153m1..14[0m
[31;1mnot ok - csv is NOT regular[0m
[32mok - header is as expected[0m
[32mok - row_count >= 100 (125)[0m
[32mok - columns.a.fill_rate == 100[0m
[32mok - columns.b.fill_rate >= 33 (34.33)[0m
[35;1mFailed plan: only 5/14 planned tests seen
[0m